src/ralph/export_to_ng/resources.py

```diff
@@ -427,8 +445,61 @@ def get_queryset(self):
         return list(chain(parents, children))
 
 
+class NetworkKindResource(resources.ModelResource):
+    class Meta:
+        fields = ('id', 'name',)
+        model = models_network.NetworkKind
+
+
+class DiscoveryDataCenterResource(resources.ModelResource):
```

Comment:
shouldn't we skip here DCs which have equivalent in assets? comment is suggesting that, but i don't see any code that is doing that